Dockerfile

```diff
@@ -29,38 +31,71 @@ ENV _CUDA_COMPAT_TIMEOUT=90
 RUN pip install git+https://github.com/romerojosh/benchy.git
 # TODO use torch-harmonics pip package after the upgrade
 RUN pip install https://github.com/NVIDIA/torch-harmonics/archive/8826246cacf6c37b600cdd63fde210815ba238fd.tar.gz
-RUN pip install "tensorly>=0.8.1" "vtk>=9.2.6" "pyvista>=0.40.1" https://github.com/tensorly/torch/archive/715a0daa7ae0cbdb443d06780a785ae223108903.tar.gz
+RUN pip install "tensorly>=0.8.1" https://github.com/tensorly/torch/archive/715a0daa7ae0cbdb443d06780a785ae223108903.tar.gz
+
+# copy modulus source
+COPY . /modulus/
+
+# install vtk and pyvista
+RUN if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
```

Comment:
Why is this `arm64` instead of `aarch64`? Similarly for `amd64` instead of `x86_64`? Would be nice to align this to the output of `uname -p` if there is no other constraint

Reply:
I think they are normalized anyways, hence I was using the normalized values directly (reference: https://github.com/containerd/containerd/blob/v1.4.3/platforms/platforms.go#L86) and their official docs used the above names. Do we need to change in that case (because it will normalize anyways)?

Reply:
If there is a precedent for defining it the way we have it, then let's leave it as is. Maybe just add a documentation item pointing to where we got this convention from.

Reply:
Added a line in the Makefile